codes/quantum/qubits/bacon_shor.yml

```diff
@@ -8,8 +8,17 @@ physical: qubits
 logical: qubits
 
 name: 'Bacon-Shor code'
```

Comment:
Please mention briefly fault-tolerance + gates from, e.g., https://arxiv.org/pdf/1911.11317.pdf and https://arxiv.org/pdf/1211.1400.pdf